charts/apisix/values.yaml

```diff
@@ -312,6 +312,11 @@ logs:
   errorLog: "/dev/stderr"
   errorLogLevel: "warn"
 
+core:
```

Comment:
Use `nginx` will be better, `core` is too generic.

Reply:
+1

Reply:
Is it possible `logs、core` node add parent node `nginx`.
`worker_rlimit_nofile` command ... on `nginx` core model.
example:
nginx:
   logs:
      enableAccessLog
   core:
      worker_rlimit_nofile:

Reply:
> Is it possible `logs、core` node add parent node `nginx`. `worker_rlimit_nofile` command ... on `nginx` core model. example:
> 
> ```
> nginx:
>    logs:
>       enableAccessLog
>    core:
>       worker_rlimit_nofile: 
> ```

good idea.

Reply:
done